x-pack/plugins/cloud_security_posture/public/pages/findings/findings_container.tsx

```diff
@@ -19,21 +19,24 @@ import { useUrlQuery } from '../../common/hooks/use_url_query';
 import { useFindings } from './use_findings';
 import type { FindingsGroupByNoneQuery } from './use_findings';
 import type { FindingsBaseURLQuery } from './types';
-import { useFindingsByResource } from './use_findings_by_resource';
+import { FindingsByResourceQuery, useFindingsByResource } from './use_findings_by_resource';
 import { FindingsGroupBySelector } from './findings_group_by_selector';
 import { INTERNAL_FEATURE_FLAGS } from '../../../common/constants';
 import { useFindingsCounter } from './use_findings_count';
 import { FindingsDistributionBar } from './findings_distribution_bar';
 import { FindingsByResourceTable } from './findings_by_resource_table';
 
-// TODO: define this as a schema with default values
-export const getDefaultQuery = (): FindingsBaseURLQuery & FindingsGroupByNoneQuery => ({
+// TODO: separate queries
+export const getDefaultQuery = (): FindingsBaseURLQuery &
+  FindingsGroupByNoneQuery &
+  FindingsByResourceQuery => ({
```

Comment:
there's a `TODO` here because currently we use 1 default query to cover both views: group by - `none` / `resource` 

this is bad because, for example, on `none` screen, URL could still have an `after` property which it will inherit from the `resource` view. 

we will need another task to define how each URL state should be handled for the entire app.

Reply:
OK 

my next task is to add sorting, and at this point i had to separate the queries. 
this is because `sort` exists in both, but each has different columns, so we don't want it to persist. 

we can still do this in a more thorough way for the entire app, but for `findings` it's not an issue anymore. i'll link to the relevant LOC once the PR is up.